automatic/qbittorrent/qbittorrent.nuspec

```diff
@@ -47,6 +47,7 @@ it for the BitTorrent protocol itself.
 ## Notes
 
 - This version includes the 64-bit version of qbittorrent, if you wish to continue using the 32-bit version you need to pass `--x86` when calling `choco install/update`
+- Beginning with v4.5.0, only 64-bit version is offered. If you have the 32-bit version installed, pin the version to 4.4.5 with command `choco pin add --name="'qbittorrent'" --version="'4.4.5'"`
```

Comment:
The description comes from the README.md which you have already updated. You don't need to update it here too.